### DISCUSSION TITLE
能否在期刊标签处增加中文期刊综合影响因子或者复合影响因子的fields

### DISCUSSION THREAD

**Discussion by wangchuanCC** [MuiseDestiny/zotero-style#288] [Ideas]
能否在期刊标签处增加中文期刊综合影响因子或者复合影响因子的fields
<img width="739" alt="image" src="https://user-images.githubusercontent.com/115852858/226628141-886437a7-a159-4ee7-8e93-719a3daece4a.png">

**Comment by wangchuanCC** [comment]
这样就省去至少两列内容，期刊标签就能更大了。

**Comment by wangchuanCC** [comment]
或者说给一个条件语句 if  语言 = zh ，影响因子 = 综合影响因子（复合影响因子），那所有文献不管中英文的影响因子都能在一个列显示了。

**Comment by viickil** [comment]
您好，请问您在信息栏的综合和复合影响因子是利用什么插件获取的呢

**Comment by MuiseDestiny** [comment]
现已支持。https://gitee.com/MuiseDestiny/plugins/blob/master/zotero-style.xpi

13 Polygon发布了一篇小红书笔记，快来看吧！ 😆 PulNuCWoC8uVfg0 😆 http://xhslink.com/a/My30PIspryvW，复制本条信息，打开【小红书】App查看精彩内容！
